src/main/java/org/prebid/server/proto/openrtb/ext/response/ExtBidResponse.java

```diff
@@ -9,7 +9,7 @@
 /**
  * Defines the contract for bidresponse.ext
  */
-@AllArgsConstructor(staticName = "of")
+@Builder
```

Comment:
Actually i don’t think this is good idea. It seems kind of breaking rules, but in this particular case we want to be sure in response object consistency.